src/realm.h

```diff
@@ -2976,6 +2977,7 @@ RLM_API realm_auth_provider_e realm_auth_credentials_get_provider(realm_app_cred
 RLM_API realm_app_config_t* realm_app_config_new(const char* app_id,
                                                  const realm_http_transport_t* http_transport) RLM_API_NOEXCEPT;
 
+RLM_API const char* realm_app_get_default_base_url(void) RLM_API_NOEXCEPT;
```

Comment:
is this supposed to be used in any way by the user?

Reply:
It is primarily provided in case the SDK wants to use the default base URL value from Core; similar to the `App::default_base_url` value.

Reply:
but what's the purpose of it? to feed it back in the app?

Reply:
Yes, it just returns a string value that the SDK (user) could supply to the `realm_app_config_set_base_url()` or `realm_app_update_base_url()` functions.